Update virtual-machines-best-practices-vm-size.md (#9814)
MicrosoftDocs/sql-docs: azure-sql/includes/virtual-machines-best-practices-vm-size.md

- Use the [Data Migration Assistant](https://www.microsoft.com/download/details.aspx?id=53595) and [SKU recommendation](/sql/dma/dma-sku-recommend-sql-db) tools to find the right VM size for your existing SQL Server workload.
- Use [Azure Data Studio](/azure/dms/tutorial-sql-server-to-virtual-machine-online-ads) to migrate to Azure.

> [!WARNING]

> Installing SQL Server to systems that exceed 64 cores per NUMA node is not currently supported, such as the Standard_M176s_3_v3 and Standard_M176s_4_v3 Azure Virtual Machine sizes within the Msv3 and Mdsv3 Medium Memory Series.

Reviewer: ```suggestion
> Installing SQL Server to systems that exceed 64 cores per NUMA node is not currently supported. This limitation currently applies to the Standard_M176s_3_v3 and Standard_M176s_4_v3 Azure Virtual Machine sizes within the Msv3 and Mdsv3 Medium Memory Series.
```

> SQL Server is not supported on machines with more than 64 cores per NUMA node. As a result, it will not be possible to provision SQL Server on systems that exceed 64 cores per NUMA node. Presently, this constraint only affects the Standard_M176s_3_v3 and Standard_M176s_4_v3 Azure Virtual Machine sizes within the Msv3 and Mdsv3 Medium Memory Series.

Reviewer: ```suggestion
Installing SQL Server to systems that exceed 64 cores per NUMA node is not currently supported, such as the Standard_M176s_3_v3 and Standard_M176s_4_v3 Azure Virtual Machine sizes within the Msv3 and Mdsv3 Medium Memory Series.
```
